samples/Makefile

```diff
@@ -12,15 +12,19 @@ SYS	= c64
 # source tree; otherwise, use the "install" directories.
 ifeq "$(wildcard ../src)" ""
 # No source tree
-MOUS = /usr/lib/cc65/mou/$(SYS)*.mou
-TGI  = /usr/lib/cc65/tgi/$(SYS)*.tgi
+MOUS = /usr/lib/cc65/target/$(SYS)/drv/mou/$(SYS)*.mou
+TGI  = /usr/lib/cc65/target/$(SYS)/drv/tgi/$(SYS)*.tgi
 ifneq "$(wildcard /usr/local/lib/cc65)" ""
-MOUS = /usr/local/lib/cc65/mou/$(SYS)*.mou
-TGI  = /usr/local/lib/cc65/tgi/$(SYS)*.tgi
+MOUS = /usr/local/lib/cc65/target/$(SYS)/drv/mou/$(SYS)*.mou
+TGI  = /usr/local/lib/cc65/target/$(SYS)/drv/tgi/$(SYS)*.tgi
+endif
+ifneq "$(wildcard /opt/local/share/cc65)" ""
```

Comment:
> [...] and eventually used the same approach that's already utilised in this Makefile

... and that's the very reason why I don't oppose to it in the context of this pull request ;-)